lib/LaTeXML/resources/XSLT/LaTeXML-structure-xhtml.xsl

```diff
@@ -644,23 +646,34 @@
   <xsl:preserve-space elements="ltx:date"/>
   <xsl:template match="ltx:date"/>
 
-  <xsl:template match="ltx:date" mode="intitle">
+  <xsl:template name="dates">
     <xsl:param name="context"/>
-    <xsl:text>&#x0A;</xsl:text>
-    <!-- Originally, html5 seemed to suggest we might use h2 here, but that is retracted-->
-    <xsl:element name="div" namespace="{$html_ns}">
-      <xsl:call-template name="add_id"/>
-      <xsl:call-template name="add_attributes"/>
-      <xsl:apply-templates select="." mode="begin">
-        <xsl:with-param name="context" select="$context"/>
-      </xsl:apply-templates>
-      <xsl:apply-templates select="//ltx:document/ltx:date/node()">
-        <xsl:with-param name="context" select="$context"/>
-      </xsl:apply-templates>
-      <xsl:apply-templates select="." mode="end">
-        <xsl:with-param name="context" select="$context"/>
-      </xsl:apply-templates>
-    </xsl:element>
+    <xsl:param name="dates" select="ltx:date"/>
+    <xsl:if test="$dates">
+      <xsl:text>&#x0A;</xsl:text>
+      <!-- Originally, html5 seemed to suggest we might use h2 here, but that is retracted-->
+      <xsl:element name="div" namespace="{$html_ns}">
+        <xsl:attribute name="class">ltx_dates</xsl:attribute>
+        <xsl:apply-templates select="." mode="begin">
+          <xsl:with-param name="context" select="$context"/>
+        </xsl:apply-templates>
+        <xsl:apply-templates select="$dates" mode="inintitle">
```

Comment:
typo? `inintitle` -> `intitle`

Reply:
Ah, yeah. Actually was intentional during debugging, but is probably just confusing now.  I've fixed it.